### PR TITLE
Add Swift Package Manager support

### PR DESCRIPTION
**Pull request by Edward-Bazko**
Hi Jens,

Thank you for a such useful and simple repository 👍
I just created a PR to support Swift Package Manager; I think lots of people will benefit from it.

### CODE REVIEW

File: UIScreenExtension.podspec [hunk 1]
```diff
@@ -13,5 +13,5 @@ Pod::Spec.new do |s|
   s.author       = { "Jens Schwarzer" => "jens.schwarzer@marchv.com" }
   s.platform     = :ios, "9.0"
   s.source       = { :git => "https://github.com/marchv/UIScreenExtension.git", :tag => s.version }
-  s.source_files = "UIScreenExtension", "UIScreenExtension/**/*.{h,swift}"
+  s.source_files = "UIScreenExtension", "UIScreenExtension/**/*.{h,swift}", "Sources/**/*.{h,swift}"
```
Comment (marchv) [inline UIScreenExtension.podspec:16]: Is this change needed? :)

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.2
+
+import PackageDescription
+
+let package = Package(
+    name: "UIScreenExtension",
+    platforms: [
+        .iOS(SupportedPlatform.IOSVersion.v9),
+        .tvOS(SupportedPlatform.TVOSVersion.v9),
+        .watchOS(SupportedPlatform.WatchOSVersion.v2)
```
Comment (marchv) [inline Package.swift]: I'm not sure about these two? :)

**Review by marchv — CHANGES_REQUESTED** [review]
Hi Edward and thanks a million! :D 

I have a few comments - please check and let me know. I'm ready to approve it once it is sorted out :)

Cheers!

File: Package.swift (hunk 2 above)
Comment (Edward-Bazko) [reply Package.swift]: Well, me neither to be honest 😳 The readme says that the extension supports tvOS and watchOS, so I added them as supported platforms to reflect the statement.

Practically, if clients have tvOS or watchOS apps they just can't use the package without these two lines.

File: UIScreenExtension.podspec (hunk 1 above)
Comment (Edward-Bazko) [reply UIScreenExtension.podspec:16]: Yes, I think so. I moved the file in the root directory:
from: `UIScreenExtension/UIScreenExtension.swift`
to: `Sources/UIScreenExtension/UIScreenExtension.swift` to conform SPM file structure.

So Cocoapods can find the extension file itself (the pod will include all `.h` and `.swift` files in `Sources` folder and subfolders if at some point more files will be added to the project).

File: Package.swift (hunk 2 above)
Comment (Edward-Bazko) [reply Package.swift]: oh, hold on, my bad, the doc says it's for improvements. And they are're not supported indeed. Will remove the lines

**Comment by Edward-Bazko** [comment]
Ouch, do not merge it please. Realized the project file is broken 🥵

**Comment by Edward-Bazko** [comment]
> Ouch, do not merge it please. Realized the project file is broken 🥵

Corrected

**Comment by marchv** [comment]
Great! And thanks a lot again! 👍

**Pull request merged**